README.md

```diff
@@ -27,8 +27,8 @@ docker-compose up
 ```
 When [`localhost:3000`](http://localhost:3000/) is up, enter this in a new terminal:
 ```
-run_migrations_in_container.py
-# TODO: Load sample data into DB
+./run_migrations_in_container.py
+./load_hca_metadata.py path_to_metadata_from_HCA.tsv
```

Comment:
Can you add a `wget` line to explain where precisely this metadata TSV comes from?

Reply:
I'm not sure -- the metadata download links on HCA's portal have some scripting functionality to prepare a download (or something like that). I don't know how stable the URLs are, so I'll add a note to download metadata with a web browser.